stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1414,4 +1414,4 @@ function normalize(v::AbstractVector, p::Real = 2)
         T = typeof(zero(eltype(v))/nrm)
         return T[]
     end
-end
+end
```

Comment:
Did you mean to delete this terminating newline?

Reply:
Honestly not too plugged into the whitespace changes. Will revert.